### DISCUSSION TITLE
Multiple cameras Stream

### DISCUSSION THREAD

**Discussion by nunokaneco** [thirtythreeforty/neolink#303] [Q&A]
Hello 

Is there any way to use in the same machine /config file for more then one camera stream?  

Like this ?? 

[[cameras]]
name = "Camera1"
username = "**********"
password = "**********"
address = "192.168.xxx.xxx:9000"

[[cameras]]
name = "Camera2"
username = "**********"
password = "**********"
address = "192.168.xxx.xxx:9000" 

Thank you for your support !

**Comment by spacebares** [comment]
```
 bind = "0.0.0.0"
 
 [[cameras]]
 name = "cam1"
 username = "admin"
 password = "admin"
 address = "192.168.x.x:9000"
 [[cameras]]
 name = "cam2"
 username = "admin"
 password = "admin"
 address = "192.168.x.x:9000"
 [[cameras]]
 name = "cam3"
 username = "admin"
 password = "admin"
 address = "192.168.x.x:9000"
```

this works for me, it does 3 cameras with 3 sub stream and 3 main.

**Comment by QuantumEntangledAndy** [comment]
Your current suggested config file should work, providing you put in the camera passwords correctly. Have you tried it? If so did you get an error. Perhaps if you posted the error we could help you more
